packages/kbn-observable/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig",
+  "compilerOptions": {
+    "declaration": true,
+    "stripInternal": true,
+    "outDir": "./target"
```

Comment:
We should add `"module": "CommonJS"` so this module doesn't rely on `babel-register` to function. Maybe even `"target": "ES2017"`?